query/query0_test.go

```diff
@@ -3484,6 +3484,38 @@ func TestMatchingWithPagination(t *testing.T) {
 	}
 }
 
+func TestKRandomNodes(t *testing.T) {
+	q := `{
+		data(func: uid(61, 62, 63, 64, 65, 66, 67, 68, 69, 70), random: 2) @filter(has(connects)) {
```

Comment:
Node 69,70 exist?

Reply:
Yes, they exist but It doesn't matter, but for clarity, I have removed them from the query.